lib/ttnt/internals.rb

```diff
@@ -0,0 +1,9 @@
+require 'rake'
+
+module TTNT
+  class << self
+    def root_dir
+      @root_dir ||= Rake.application.find_rakefile_location[1]
```

Comment:
Sounds legit.

My understanding is that class variable is better since we want `root_dir` to be consistent across application, and don't want other classes to have `root_dir` (class) instance variables with different content in cases like this class is inherited (though it will not happen).
Is this correct?

Anyway thank you for pointing out! This choice (to express what we want the variable to mean) is quite interesting topic for me :smile: 
I changed to use class instance variable here: https://github.com/Genki-S/ttnt/commit/8d226b0acde3271b6ac712c3dae6c5c7e77c9b04